typescript/005-resistor-color-trio/resistor-color-trio.ts

```diff
@@ -32,18 +32,14 @@ export function decodedResistorValue([color1, color2, color3]: Colors[]): string
 		exp += 1;
 	}
 
-	let finalExp = exp;
-	let finalSuffix = "ohms";
-
-	for (const [key, value] of Object.entries(SI_PREFIXES)) {
-		if (exp >= Number(key)) {
-			finalExp = exp - Number(key);
-			finalSuffix = `${value}ohms`;
+	for (const [key, value] of SI_PREFIXES) {
```

Comment:
Notice how you don't need neither `Object.entries` now nor the typecasting. This is the correct approach.